### PR TITLE
updates learning goals for 2011

### PR DESCRIPTION
**Pull request by BrianZanti**

### CODE REVIEW

File: module2/success/learning_goals.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Module 2
 subheading: Learning Goals
 ---
 
-In addition to our [guidlines and expectations](./guidelines_and_expectations), academic success in Module 2 means that students demonstrate proficiency and comfort with the concepts below.
+Academic success in Module 2 means that students demonstrate proficiency and comfort with the concepts below.
```
Comment (BrianZanti) [inline module2/success/learning_goals.md:7]: I decided to take out this "Guidelines and Expectations" page. You can view the original page [here](https://backend.turing.io/module2/success/guidelines_and_expectations). I like everything that is said in that page, but I feel like it is not appropriate for this to be a mod 2 specific page. For example, the section about "Attendance" gets outdated as we update the school-wide attendance policy. We could consider rolling this info into our intro to mod 2 slides if we think there's anything from this Guidelines and Expectations doc we'd like to keep.

File: module2/success/learning_goals.md [hunk 2]
```diff
@@ -14,36 +14,34 @@ The expected mastery level can be understood with the following scale:
 
 # Mastery
```
Comment (BrianZanti) [inline module2/success/learning_goals.md:15]: Does everything in the Mastery section accurately represent what a student needs to know coming out of Mod 2?

File: module2/success/learning_goals.md [hunk 3]
```diff
@@ -53,71 +51,80 @@ The expected mastery level can be understood with the following scale:
...
 
-* Implement partials to break a page into reusable components
-* Make use of flash messages
-* Use Sessions to store information about a user and implement login/logout functionality
-* Use filters (e.g. `before_action`) in a Rails controller
-* Use Inheritance from ApplicationController or a student created controller to store methods that will be used in multiple controllers
-* Limit functionality to authorized users
-* Use BCrypt to hash user passwords before storing in the database
-* Use POROs to logically organize code for objects not stored in the database
+# Functional
```
Comment (BrianZanti) [inline module2/success/learning_goals.md:76]: Does everything in the Mastery and Functional sections accurately represent what a student will be doing in their projects throughout mod 2?

File: module2/success/learning_goals.md (hunk 1 above)
Comment (timomitchel) [reply module2/success/learning_goals.md:7]: That makes sense to me.

File: module2/success/learning_goals.md (hunk 2 above)
Comment (timomitchel) [reply module2/success/learning_goals.md:15]: Yea, I think that covers it.

File: module2/success/learning_goals.md [hunk 4]
```diff
@@ -53,71 +51,80 @@ The expected mastery level can be understood with the following scale:
...
 
-#### ActiveRecord
+### Web Applications
 
-* Use built-in ActiveRecord methods to join multiple tables of data, calculate statistics and build collections of data grouped by one or more attributes
+* Describe the HTTP request/response cycle
```
Comment (timomitchel) [inline module2/success/learning_goals.md:80]: I like moving this here. We never test them on this, but we do expect them to understand it.

**Review by timomitchel — APPROVED** [review]
I like these changes. Thanks for looking into this.

**Pull request merged**